tools/scripts/build_projects.py

```diff
@@ -192,14 +192,17 @@ def build(self):
 				new_hdf = True
 		
 		if new_hdf:
-			err = run_cmd(cmd + ' clean_all')
+			err = run_cmd(cmd + ' reset')
```

Comment:
the changes in build_projects should probably go in the 2nd commit, that's the one that deals with reset and update

Reply:
Done